jest.setup.js

```diff
@@ -12,8 +12,3 @@ process.env.NEXT_PUBLIC_WALLET_URL = 'https://wallet-calibration.glif.link'
 process.env.NEXT_PUBLIC_SAFE_URL = 'https://safe-calibration.glif.link'
 process.env.NEXT_PUBLIC_EXPLORER_URL = 'https://explorer-calibration.glif.link'
 process.env.NEXT_PUBLIC_VERIFIER_URL = 'https://verify-calibration.glif.link'
```

Comment:
Can we take the URLs from the TestEnvironment as well instead of supplying them here?